### PR TITLE
Tribs: add commons

### PR DESCRIPTION
**Pull request by nickhill111**
SSCS-11339

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierAboutToSubmitHandlerTest.java [hunk 1]
```diff
@@ -84,6 +85,7 @@ void givenAInvalidCallbackType_thenReturnFalse() {
     @ParameterizedTest
     @EnumSource(SendToFirstTierActions.class)
     void shouldReturnWithoutError(SendToFirstTierActions action) {
+        caseData.setHearings(List.of(Hearing.builder().value(HearingDetails.builder().build()).build()));
```
Comment (DGordonV1) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierAboutToSubmitHandlerTest.java:88]: Ideally we don't want to be using nested builders, cleaner to separate them out even if no params being set/passed.

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java [hunk 2]
```diff
@@ -83,4 +85,50 @@ void shouldReturnWithoutError() {
...
+            .build()));
+
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(caseData);
+
+        PreSubmitCallbackResponse<SscsCaseData> response =
+                handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+
+        assertThat(response.getErrors()).isEmpty();
+        assertThat(response.getData().getSchedulingAndListingFields().getPanelMemberExclusions().getExcludedPanelMembers().contains(new CollectionItem<>("", judge))).isTrue();
```
Comment (DGordonV1) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java:109]: Could probably chop this on to new lines and the one below.

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java [hunk 3]
```diff
@@ -37,17 +39,26 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
                                                           String userAuthorisation) {
         SscsCaseData caseData = callback.getCaseDetails().getCaseData();
 
-        PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(caseData);
-
         PostHearing postHearing = caseData.getPostHearing();
         log.info("Review Post Hearing App: handling action {} for case {}", postHearing.getReviewType(), caseData.getCcdCaseId());
 
-        if (response.getErrors().isEmpty()) {
```
Comment (DGordonV1) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java:45]: Do we still want to check response for errors first here? Why not?

Comment (nickhill111) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java:45]: there's no possibility of having an error before this

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierAboutToSubmitHandlerTest.java (hunk 1 above)
Comment (nickhill111) [reply src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierAboutToSubmitHandlerTest.java:88]: done

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java (hunk 2 above)
Comment (nickhill111) [reply src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java:109]: done

**Review by DGordonV1 — DISMISSED** [review]

**Review by paul-pearson — DISMISSED** [review]

**Review by AndreiTrushkov — APPROVED** [review]

**Review by DGordonV1 — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**